release-notes.md

```diff
@@ -7,16 +7,18 @@ been released.
 
 ## Future Plans
 
-As a reminder, TimescaleDB is open-source and you
-can view active developments on GitHub at any time.
+TimescaleDB is an open source project with a vibrant community. 
+We are currently focusing on making our priorities known by that community and you are free to visit us on our Github repo and meet us on our [community Slack](https://timescaledb.slack.com).
 
 ### What to expect from our next release
 
-The team is finishing up work on the first release of distributed hypertables
-supported by multi-node TimescaleDB, which will be part of the TimescaleDB 2.0
-release.  We are currently in the middle of formal Release Candidates (RC) for
-2.0; these RC releases support in-place upgrades from 1.x and subsequent
-upgrades to 2.0.
+From our next release (2.0.1) we are planning to bring:
+- [Support for PostgreSQL 13](https://github.com/timescale/timescaledb/issues/2779): PG13 was released on September 24, 2020 and offers signifficant improvements in performance and usability. We want to provide our users the latest and greatest versions of PG.
+- Backup and restore for multi-node deployments so our users can restore their data to a consisstent state across all nodes.
+- Adding and renaming columns to compressed hypertables as a step towards advancing our suport for compressed hypertables.
+- As well as some bug fixes.
```

Comment:
If we announce it this way we are (indirectly) committing to deliver features in a specific release, which means that we have to leave the date open (for our release) in case we have delays. This contrasts with "we just deliver what we have at this date", which I think is fine. Users are more interested in what we plan for the next minor/major release rather than committing to a specific date.

I think we can remove the "as well as some bug fixes". It goes without saying.

```suggestion
From our next release (2.0.1) we are planning to bring:
- [Support for PostgreSQL 13](https://github.com/timescale/timescaledb/issues/2779): PG13 was released on September 24, 2020 and offers significant improvements in performance and usability. We want to provide our users the latest and greatest versions of PG.
- Backup and restore for multi-node deployments so our users can restore their data to a consistent state across all nodes.
- Adding and renaming columns to compressed hypertables as a step towards advancing our support for compressed hypertables.
- As well as some bug fixes.
```